samples/core/xgboost_training_cm/README.md

```diff
@@ -13,6 +13,8 @@ or not.
 ## Requirements
 
 Preprocessing uses Google Cloud DataProc. Therefore, you must enable the [DataProc API](https://cloud.google.com/endpoints/docs/openapi/enable-api) for the given GCP project.
+If KFP was deployed through K8S marketplace, please follow instructions in [the guideline](https://github.com/kubeflow/pipelines/blob/master/manifests/gcp_marketplace/guide.md#gcp-service-account-credentials)
+to make sure the service account used has the role `storage.admin` and `dataproc.admin`.
```

Comment:
Probably also add the API enabled requirement, such as "enable the [DataProc API](https://cloud.google.com/endpoints/docs/openapi/enable-api) for the given GCP project"

Reply:
We can have a direct link to enabling that API.

Reply:
Add a link to the API page.
@Ark-kun are you saying there is a direct link that can enable the API within one-click? If so that'll be better.

Reply:
When you try running a Dataproc or Dataflow component, that link is in the error log.

Here is the Dataflow link: https://console.cloud.google.com/apis/api/dataflow.googleapis.com/overview?project=1234567